tests/lib/rules/max-len.js

```diff
@@ -309,6 +314,17 @@ ruleTester.run("max-len", rule, {
                 }
             ]
         }, {
+            code: "// A comment that exceeds the max comment length, but not the specified max length for code.",
+            options: [40, 4, { comments: 80 }],
```

Comment:
The code can be 40, but comments can be 80.  Prior to this PR, the comment in this test would have failed because it exceeds the limit for code (even though it is shorter than the specified length for comments).  The purpose of this PR is to allow the length specified for comments to rule for comment lines.